### PR TITLE
New met skims

### PR DESCRIPTION
**Pull request by MatthewDittrich**

### CODE REVIEW

**Comment by MatthewDittrich** [comment]
Skims have been checked with previous yields (turning JER off) and all looks good

File: analysis/wwz/yld_dicts_for_comp.py [hunk 1]
```diff
@@ -129,11 +129,11 @@
...
...130226136859406, None], 'sr_4l_bdt_of_5': [1.3666044164896425, None], 'sr_4l_bdt_of_6': [0.6232281068805007, None], 'sr_4l_bdt_of_7': [0.7315772419657074, None], 'sr_4l_bdt_of_8': [0.13974403824431153, None], 'sr_4l_bdt_sf_coarse_1': [0.6908862141943756, None], 'sr_4l_bdt_sf_coarse_2': [0.1604480380584667, None], 'sr_4l_bdt_sf_coarse_3': [0.6246291564151731, None], 'sr_4l_bdt_sf_coarse_4': [0.10095747608375848, None], 'sr_4l_bdt_of_coarse_1': [0.9281223300154253, None], 'sr_4l_bdt_of_coarse_2': [0.7145246058008736, None], 'sr_4l_bdt_of_coarse_3': [1.468676973392012, None], 'sr_4l_bdt_of_coarse_4': [0.31065286051751434, None], 'cr_4l_btag_of': [0.1700859022619126, None], 'cr_4l_btag_sf_offZ_met80': [0.10918164734750543, None], 'cr_4l_sf': [0.0109902711433886, None], 'sr_sf_all_cutbased': [0.8643906542489347, None], 'sr_of_all_cutbased': [1.593809866056569, None], 'sr_all_cutbased': [1.81311916108461, None], 'sr_sf_all_bdt': [1.0884175714735684, None], 'sr_of_all_bdt': [2.100901252616079, None], 'sr_all_bdt': [2.3661020441088816, None], 'sr_sf_all_bdtCoarse': [0.9504847862547728, None], 'sr_of_all_bdtCoarse': [1.904067678337036, None], 'sr_all_bdtCoarse': [2.1281200747583706, None]}}
```
Comment (kmohrman) [inline analysis/wwz/yld_dicts_for_comp.py:135]: Can you update the y22 and y23 numbers too?

**Comment by MatthewDittrich** [comment]
Note that with the randomness associated with JER, yield comparisons on new skims can only tested effectively with JER omitted. See table below for the yield comparison change when JER is left on:

Screen Shot 2024-10-11 at 10 38 07 AM

**Comment by kmohrman** [comment]
Looks good. One last request before we merge. Could you run these through the card making steps and report the new ref numbers?

**Comment by MatthewDittrich** [comment]
New (previous) refs are:
R2: 4.57772 (4.57517)
R3: 2.48115 (2.49108)
Comb: 5.13394 (5.12477)

**Pull request merged**